client/state/jetpack/modules/actions.js

```diff
@@ -145,10 +144,16 @@ export const fetchModuleList = ( siteId ) => {
 				path: '/jetpack/v4/module/all/',
 			} )
 			.then( ( { data } ) => {
-				const modules = mapValues( data, ( module ) => ( {
-					active: module.activated,
-					...omit( module, 'activated' ),
-				} ) );
+				const modules = Object.entries( data ).reduce(
+					( acc, [ key, { activated: active, ...module } ] ) => {
+						acc[ key ] = {
+							active,
+							...module,
+						};
+						return acc;
+					},
+					{}
+				);
```

Comment:
Instead of `reduce`, I think using `entries |> map \> fromEntries` would be a better pattern to use?
```js
Object.fromEntries(
  Object.entries().map( ( [ key, value ] ) => {
    const { activate: active, ...module } = value;
    return [ key, { active, ...module } ];
  } )
);
```
What do you think?